src/Uno.Extensions.Maui.UI/MauiHost.cs

```diff
@@ -140,6 +148,8 @@ element switch
 		};
 #endif
 
+	public new VisualElement? Content { get; private set; }
```

Comment:
Instead of 'new' (🤮), would it be more obvious if it was a different property that made it clear it is the maui visual. Like `MauiVisual` or something.

Reply:
I agree, re-purposing Content is confusing. I'd suggest `MauiContent` as the property name